test.properties

```diff
@@ -1576,7 +1571,7 @@
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8]=\
         (app.server.types == null OR app.server.types ~ tomcat) AND \
         (database.types == null OR database.types ~ mysql) AND \
-        (portal.acceptance == true)
+        (portal.acceptance == john-co)
```

Comment:
I guess this answers my question above and that these changes should not be committed and are here for test purposes only...

I'll change this to Draft to make sure we are more aware that is in need of additional changes.